wovn_index_sample.php

```diff
@@ -12,13 +12,33 @@
   "index.phtml",
   "app.php"
 );
-$paths = wovn_helper_detect_paths(dirname(__FILE__), parse_url($_SERVER["REQUEST_URI"], PHP_URL_PATH), $files);
-# SSI USER: please swap comments on the two lines below
-$included = wovn_helper_include_by_paths($paths);
-# $included = wovn_helper_include_by_paths_with_ssi($paths);
+
+$parsed_url = parse_url($_SERVER["REQUEST_URI"], PHP_URL_PATH);
+
+if ($parsed_url) {
+  $paths = wovn_helper_detect_paths(dirname(__FILE__), $parsed_url, $files);
+  
+  # SSI USER: please swap comments on the two lines below
+  # (also see the SSI comment in the 404 section below)
+  $included = wovn_helper_include_by_paths($paths);
+  # $included = wovn_helper_include_by_paths_with_ssi($paths);
+}
```

Comment:
The whitespace on lines 19 and 23 causes the SSI test to fail.  If those lines are flush with 'if' (they are untabbed), then the test passes.